scripts/jenkins/ardana/ansible/roles/input_model_generator/defaults/main.yml

```diff
@@ -20,6 +20,9 @@ qe_ardana_bm_info_dir: "/tmp/ardana-bm-info"
 
 template_file_name: "{{ scenario[item ~ '_template'] }}.yml"
 
+# Backend for designate: powerdns or bind
+designate_backend: bind
```

Comment:
Do we document somewhere that "designate_backend" (or similar?) is a setting that can be overridden via a command line setting or an included vars setting?

Reply:
I added some lines to the input model generator documentation to reflect that.